packages/composables/src/internalHelpers/defaultApiParams.json

```diff
@@ -89,15 +93,16 @@
         "total",
         "aggregations"
       ],
-      "property_group": ["id", "translated", "options"],
+      "property_group": ["id", "translated", "name", "options"],
```

Comment:
Same here?

Reply:
We also don't need the name, if we have the `translated`.

Reply:
right